algosdk/abi/tuple_type.py

```diff
@@ -71,7 +71,7 @@ def _find_bool(type_list, index, delta):
         return until
 
     @staticmethod
-    def _parse_tuple(s):
```

Comment:
nit: I think this can still be an underscore method, since it's only supposed to be used internally

Reply:
I decided to expose it so it can be used in the `Method` class to parse the tuple string: https://github.com/algorand/py-algorand-sdk/blob/d63b4d132a0b8c3aeb2b9001a3e87f596eb34423/algosdk/abi/method.py#L110

Reply:
Right it can still be used there, but unless we want consumer of the library to use that method, it's still probably best for it to remain underscored.

Reply:
Good point, I agree with that